docs/release-notes.md

```diff
@@ -19,21 +19,27 @@ With the introduction of HTTPProxy, IngressRoute CRD is now marked as deprecated
 
 The IngressRoute CRD will be supported in its current state until the Contour 1.0.0 release and will be removed shortly after.
 
-### CLF logging
+### Envoy logging
```

Comment:
s/Logging changes

By default Envoy emits request logs in its own format. See ... for details

Contour 1.0.0-beta.1 adds support for JSON formatted logs. To enable this ... do this ... 

See [documentation] and [design doc] for more infomation.